scripts/globals/survival_guide.lua

```diff
@@ -155,12 +155,12 @@ xi.survivalGuide.onEventFinish = function(player, eventId, option)
                 local teleportCostGil = SURVIVAL_GUIDE_TELEPORT_COST_GIL
                 local teleportCostTabs = SURVIVAL_GUIDE_TELEPORT_COST_TABS
 
-                -- If the player has the rhapsody in white, the cost is 10% of original gil or 20% of original tabs.
-                -- GIL: 1000 -> 200
```

Comment:
realized right after hitting the button: between cities its 100, if I jump to to a non city like say a bcnm entrance, its 200. This may be where the discrepancy came from.

Reply:
May be a matter of incorrect base values. I'll check retail, since i dont have the KI

Reply:
It's not. The base cost is indeed 1000 gil